src/test/java/com/github/commonsrdf/dummyimpl/TestWritingGraph.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.commonsrdf.dummyimpl;
+
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.github.commonsrdf.api.BlankNode;
+import com.github.commonsrdf.api.IRI;
+
+public class TestWritingGraph {
+
+	/** Run tests with -Dkeepfiles=true to inspect /tmp files **/
+	private static boolean KEEP_FILES = Boolean.getBoolean("keepfiles");
```

Comment:
Yes, I like looking at the files.. but don't want to fill up with 100x8 MB on every build :) (just 2x8MB here)

If I do a lot of them I usually make a createTempFile() method in the base test class that does this kind of logic. (Is `keepfiles` good enough name?)